src/mmw/js/src/compare/views.js

```diff
@@ -0,0 +1,168 @@
+"use strict";
+
+var _ = require('lodash'),
+    $ = require('jquery'),
+    L = require('leaflet'),
+    Marionette = require('../../shim/backbone.marionette'),
+    compareWindowTmpl = require('./templates/compareWindow.html'),
+    compareScenariosTmpl = require('./templates/compareScenarios.html'),
+    compareScenarioTmpl = require('./templates/compareScenario.html'),
+    compareMapTmpl = require('./templates/compareMap.html'),
+    compareChartTmpl = require('./templates/compareChart.html'),
+    compareModificationsTmpl = require('./templates/compareModifications.html');
+
+var CompareWindow = Marionette.LayoutView.extend({
+    //model: modelingModels.ProjectModel,
```

Comment:
I think I like this approach better than having it uncommented, like it is here: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/modeling/views.js#L265